package.json

```diff
@@ -6,9 +6,9 @@
     "electron-prebuilt": "^1.2.7"
     },
    "scripts": {
-    "lint" : "eslint *.js", 
-    "lintinstall" : "npm install eslint",
-    "pretest" : "npm run lintinstall && npm run lint"
+    "lint": "eslint *.js",
+    "pretest": "npm run lint"
+    "start" : "./node_modules/.bin/electron ."
```

Comment:
nit: extra space before the colon.

I think this can just be `"electron ."`, try it first though